templates/status.html

```diff
@@ -0,0 +1,53 @@
+<!doctype html>
+<html>
+    <head>
+        <link href="../../bootstrap.min.css" rel="stylesheet">
+    </head>
+
+    <body>
+        <div class="container">
+            <div class="page header">
+	        <h2 id="cid">Container: {{ cid }}</h2>
+                <p id="status">{{ status }}</p>
+            </div>
+            <div class="row">
+	        <div class="col-md-6">
+                    <h4>StdOut:</h4>
+		    <pre class="pre-scrollable" id="stdout"> {{ out }} </pre>
+	        </div>
+		<div class="col-md-6">
+                    <h4>StdErr:</h4>
+		    <pre class="pre-scrollable" id="stderr">{{ err }} </pre>
+	        </div>
+            </div>
+        </div>
+        <script>
+            (function() {
+              var url = '{{ url }}'
+              var checkContainer = function(url) {
+                 xmlhttp=new XMLHttpRequest();
```

Comment:
Initialize variable: `var xmlhttp = new XMLHttpRequest();` otherwise you will have a memory leak.